### PR TITLE
added high mass HppWW samples

### PR DESCRIPTION
**Pull request by jsalfeld**

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/DoublyChargedHiggsGMmodel_HWW_M2000/DoublyChargedHiggsGMmodel_HWW_M2000_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+#*                                                          *
+#*     run as ./bin/mg5_aMC  filename                       *
+#*                                                          *
+#************************************************************
+
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set gauge unitary
+set complex_mass_scheme False
+import model sm
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/DoublyChargedHiggsGMmodel_HWW_M2000/DoublyChargedHiggsGMmodel_HWW_M2000_proc_card.dat]: this is overriden by
import model GM_UFO
please remove it.
btw, is the model supporting 4F PDF?

**Comment by perrozzi** [comment]
ping

**Comment by jsalfeld** [comment]
lines are removed.

>>btw, is the model supporting 4F PDF?
I am not sure about this

**Comment by perrozzi** [comment]
don't know what to say then, please ask the authors of the UFO model... anybody having a clue?

**Comment by jsalfeld** [comment]
We specify p= u d c s (g), checked with e.g. "263400    = lhaid" and produced events, puzzled by what you mean by "supported". btw what is going on with:
https://cms-pdmv.cern.ch/mcm/mccms?prepid=HIG-2017May17-00004&page=0&shown=5239

**Comment by perrozzi** [comment]
I hope you know what you are doing...

**Pull request merged**